src/Queue/QueueBase.php

```diff
@@ -117,7 +117,7 @@ protected function getQueue(AMQPChannel $channel, array $options = []) {
 
     // Allow modules to override queue settings.
     if (isset($queue_info[$this->name])) {
-      $queue_options += $queue_info[$this->name];
```

Comment:
I'm not very good with tests yet so it'll take a bit more, but I want to also pass the Drupal coding standards in the tests.